### ISSUE TITLE
How to understand alpha in eqn.5?

### ISSUE THREAD

**Issue by Pixie8888** [RozDavid/LanguageGroundedSemseg#26]
Hi,

Thanks for releasing code. I have a question about alpha in eqn.5.  Based on my understanding, the more frequently appearing class will get higher alpha. Then it has higher weight in the focal loss. Is this contradictory to what the paper trying to do? I thought less frequently appearing class would get higher alpha, so that model pay more attention to them during optimization.

I also have a question for fine-tuning. Do you train a new classifier in the fine-tuning or using class name feature as the classifier?
